src/moh_data/main.py

```diff
@@ -1,4 +1,7 @@
 import os
+from datetime import datetime
+
+import numpy as np
 
 from .core.collector import DataCollector
 from .utils.visualisation import Visualisation
```

Comment:
FWIW, this is causing me problems when trying to run this file using `python main.py`. If I remove the leading `.` then everything is fine.